### DISCUSSION TITLE
Missing google search console settings tab

### DISCUSSION THREAD

**Discussion by simonhackler** [towfiqi/serpbear#208] [Q&A]
I want to enable google search console integration but I am missing the settings tab.
Is there something I still have to toggle somewhere?

![image](https://github.com/towfiqi/serpbear/assets/43748576/1a262bb9-354c-494c-9036-65b113c3a717)

**Comment by pavelstianko** [comment]
I have the same problem. Any solution to this?

**Comment by towfiqi** [comment]
You are probably using a very old version of SerpBear. Please update to the latest version. 

If you've installed it locally, update running this commands:

```
git pull
npm install
npm run build
npm run start:all
```

If you are using docker. Use this commands:

```
docker compose pull
docker compose up
```
